### DISCUSSION TITLE
3.1.0rc2

### DISCUSSION THREAD

**Discussion by michael-s-molina** [apache/superset#26240] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 3.1.0.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/3.1.0rc2/

Git tag for the release:
https://github.com/apache/superset/tree/3.1.0rc2

The Change Log for the release:
https://github.com/apache/superset/blob/3.1.0rc2/CHANGELOG.md

The difference between RC1 and RC2:
https://github.com/apache/superset/compare/3.1.0rc1...3.1.0rc2

The shared testing environment:
http://35.92.145.144:8080/

The Updating instructions for the release:
https://github.com/apache/superset/blob/3.1.0rc2/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/3.1.0rc2'>3.1.0rc2</a>.</em>

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/26242

**Reply by john-bodley** [reply]
Fix: https://github.com/apache/superset/pull/26243

**Comment by sfirke** [comment]
#26254

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26269

**Comment by sfirke** [comment]
#26255

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26264

**Comment by sfirke** [comment]
#26256

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26264

**Comment by gaurav7261** [comment]
https://github.com/apache/superset/issues/26276

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26284

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/26248

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26279

**Comment by sadpandajoe** [comment]
#26298

**Reply by michael-s-molina** [reply]
Moved to [RC3 discussion](https://github.com/apache/superset/discussions/26296).
